_pages/production.md

```diff
@@ -69,39 +69,45 @@ Please reach out to your agency IAA contact if you have any questions. If your a
+1.  Select the Next button to move to the next steps.
 
-5.  Next, choose an agency team for the app from the drop down menu.
+1.  Select the authentication protocol that will be used by the service provider:
 
-6.  Select the authentication protocol that will be used by the service provider. 
+    - OpenID Connect Private Key JWT
+    - OpenID Connect PKCE
+    - SAML
 
-7.  Choose the level of service and default Authentication Assurance Level (AAL) field from the drop down menus and select the correct attribute bundle. 
+1.  Select the Identity Assurance Level (IAL), default Authentication Assurance Level (AAL), and the correct attribute bundle. Select the Next button to move on to the Issuer.
 
-8.  Enter the issuer, a unique string to identify the app in the Identity Provider (IdP). 
+1.  Enter the issuer, a unique string to identify the app in the Identity Provider (IdP). Select the Next button to move on to the next step. 
 
-9.  Upload your logo and public certificate file using the "Choose a file" buttons. Note: the public certificate file may not be required if you are using the OIDC PKCE flow.
+1.  Upload your logo file and public certificate file. Note: the public certificate file may not be required if you are using the OIDC PKCE flow.
 
-10. Enter the push notification URL and redirect URIs, if applicable. Please note that your redirect URIs must be secure (HTTPS) in order to be deployed to production. You can enter additional redirect URIs by selecting the “Add another URI” button.
+1. If you selected the OIDC protocol, enter the push notification URL and redirect URIs, if applicable. Please note that your redirect URIs must be secure (HTTPS) in order to be deployed to production. You can enter additional redirect URIs by selecting the “Add another URI” button.
+  - If you selected the SAML protocol, you will be prompted to enter the Assertion Consumer Service URL, Assertion Consumer Logout Service URL, SP Initiated Login URL, SAML Assertion Encryption, Signed Response Message Requested, Return to App URL, Push notification URL, and Redirect URIs.
+  - Select the Next button to move on to the final step.
 
-11. Specify the sign-in, sign-up, and forgot password help text users will encounter in your app. This step is optional but encouraged to ensure better usability. Take a look at the [Partner Support Help Desk](https://zendesk.login.gov) for a good example of help text.    
+1. Select the sign-in, sign-up, and forgot password help text users will encounter in your app. This step is optional but encouraged to ensure better usability. Take a look at the [user experience guidance]({% link _pages/user-experience/help-text.md %}) for a good example of help text.    
 
-12. Once all fields are complete select the "Create app" button.
+1. Once all fields are complete select the "Create app" button.
```

Comment:
[praise] As a keyboard user, I'm happy we're using the word "select" instead of "click".